ansible/group_vars/all/defaults.yaml

```diff
@@ -1,6 +1,5 @@
 ansible_user: devops
 ansible_python_interpreter: /usr/bin/python3
-devnet_name: template
```

Comment:
devnet name is pectra

Reply:
i dont see where its used, so decided to remove it